tools/bridge/bridge/events.py

```diff
@@ -13,6 +14,18 @@ class Event(ABC):
 Event.register(AttributeDict)
 
 
+class ChainRole(Enum):
+    home = "home"
```

Comment:
Usually enum entries are `UPPER_CAPS` in Python, no?

Reply:
I don't know. I think it's ugly, but I don't dare to look it up.